samples/auth/ClientSignedToken.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * ClientSignedToken
+ *
+ * Create a client-signed-token and access fabric resources.
+ *
+ * Using a private key:
+ *   - export PRIVATE_KEY - set to the private key of an account with
+ *                          access to the content object.
+ *
+ * Using a custodial wallet user:
+ *   - export ID_TOKEN - set to the JWT id token obtained from the
+ *                       open id sign in
+ *
+ */
+const { ElvClient } = require("../../src/ElvClient");
+
+const networkName = "demo"; // "main" or "demo"
+
+const contentHash = "hq__93SK4rgxMarq1ZeDSEu9WJkDoptTKYiA2GmYocK7inMthUssGkG6Q9BREBEhNtVCiCBFsPd4Gd";
+
+const SetupWithCustodialWallet = async () => {
+
+  client = await ElvClient.FromNetworkName({networkName});
+  await client.SetRemoteSigner({idToken: process.env.ID_TOKEN, unsignedPublicAuth: true})
+  client.ToggleLogging(false);
+
+  return client;
+}
+
+const SetupWithPrivateKey = async () => {
+
+  client = await ElvClient.FromNetworkName({networkName});
+  let wallet = client.GenerateWallet();
+  let signer = wallet.AddAccount({
+    privateKey: process.env.PRIVATE_KEY
+  });
+  client.SetSigner({signer});
+  client.ToggleLogging(false);
+
+  return client;
+}
+
+const MakeClientSignedToken = async ({client}) => {
+
+  const token = await client.CreateFabricToken({
+	  duration: 60 * 60 * 1000, // millisec
+    addEthereumPrefix: false,
+  });
+
+  console.log("ETH_TOKEN", token);
+  return token;
+}
+
+const MakeClientSignedTokenPersonal = async ({client}) => {
+
+  const token = await client.CreateFabricToken({
+    duration: 60 * 60 * 1000, // millisec
+    addEthereumPrefix: true,
```

Comment:
@elv-serban Not sure if this was intentional, but to use the `personal_sign` path, `addEthereumPrefix` should be set to `false`. `true` uses the bare `eth_sign` path

Reply:
Changes look good, thanks